_docs/vision.md

```diff
@@ -15,39 +15,39 @@ PostgreSQL?
 
 Welcome to Babelfish. The idea of the project is to close the gap between
 PostgreSQL and other systems by making PostgreSQL speak other protocols such as
-TDS, and maybe a lot more in the future. By implementing "MS SQL"-slang, we make
+TDS, and maybe a lot more in the future. By implementing "Microsoft SQL Server"-slang, we make
 it easier for people to move to PostgreSQL. 
 
 However, moving applications is just one side of the equation. Migrating data
 is one thing - making sure that applications work properly is another, far
 more important thing. In fact, migrating data to PostgreSQL can be relatively
-easy, while migration entire application stacks can be really difficult. 
+easy, while migrating entire application stacks can be really difficult.
 
 That's exactly why Babelfish is so important. It allows you to take existing
 applications, hook them up to TDS-enabled PostgreSQL and run them more quickly
 and more easily, even while some migration work might still be going on. 
 
-A second option is to move application from MS SQL to PostgreSQL without
+A second option is to move application from Microsoft SQL Server to PostgreSQL without
 touching the application at all. Why is that so important? If you get your
 source code under control, life is usually easier. But what if you want to move
 old legacy applications from Microsoft SQL Server to PostgreSQL, legacy applications which cannot be
 modified easily, because the source code is not open, is lost, or is simply too hard to
 change? Babelfish is an excellent vehicle to help in this case as well. You can
-move away from expensive MS SQL deployments without doing a painful app
+move away from expensive Microsoft SQL Server deployments without doing a painful app
 migration. Brilliant, isn't it?
 
```

Comment:
This last paragraph is a little disorganized and makes some bold assertions (the customers source code is out of control, their legacy app will work when connected out of the box, etc).  How about:

If you are using a legacy application, Babelfish may provide an easy way to move away from lost source code or an application that is no longer maintainable for technical or financial reasons. The visibility and cost savings offered by open-source software projects (like PostgreSQL with Babelfish) can be significant.

Reply:
The text makes no assertions whatsoever. It says "What if" to present a quite probable scenario.
I think that simply making the language duller is no improvement.

Reply:
Duller is better in tech docs. The sort of language Susan refers to is more fitting with sales/marketing material than with tech docs. We should stay away from that, as that belong in other places.
If we want to refer to 'source code being out of control' (what do these words mean precisely, BTW?), then it has to be in the context of a specific use case where out product/solution applies.

Reply:
I guess the problem here is that the text assumes some familiarity with software development &ndash; for a developer, "source code gone out of control" does not need an explanation.
But you are right; the target audience are not necessarily people who are familiar with development.

I had to remove some inaccuracies from Susan's suggestion (Babelfish does *not* enable you to move away from an old application, using open source offers no visibility), and ended up with b7b4431a96a3855509e2f553221a79aa9dce5379.

Is that acceptable?